rust/private/rustc.bzl

```diff
@@ -208,7 +208,7 @@ def rustc_compile_action(
     args.add("--emit=dep-info,link")
     args.add("--color", "always")
     args.add("--target", toolchain.target_triple)
-    args.add_all(ctx.attr.crate_features, before_each = "--cfg", format_each = "feature=%s")
+    args.add_all(ctx.attr.crate_features, before_each = "--cfg", format_each = "feature=\"%s\"")
```

Comment:
nit; a little easier to read as 'feature="%s"'

This quoting is necessary because features can have spaces in them?